Documentation/developer-setup-linux.md

```diff
@@ -9,52 +9,67 @@ After compiling Unreal Engine, set the following environment variables in your `
 
 ## CMake command-line
 
-First, configure the CMake project in the `~/dev/cesium-unreal-samples/Plugins/cesium-unreal/extern` directory by following the instructions below.
+First, configure the CMake project in the `~/dev/cesium-unreal/extern` directory by following the instructions below.
 **Note**: The following steps must be done in the `extern` directory, and *not* the `cesium-native` subdirectory!
 
-Change to the `~/dev/cesium-unreal-samples/Plugins/cesium-unreal/extern` directory, and execute the following commands to build and install a Debug version of cesium-native:
+Change to the `~/dev/cesium-unreal/extern` directory, and execute the following commands to build and install a Debug version of cesium-native:
 
-      cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Debug
-      cmake --build build --target install
+    cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Debug
+    cmake --build build --target install -j14
 
 To build a Release version, do the following:
 
-      cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Release
-      cmake --build build --target install
+    cmake -B build -S . -DCMAKE_TOOLCHAIN_FILE="unreal-linux-toolchain.cmake" -DCMAKE_POSITION_INDEPENDENT_CODE=ON -DCMAKE_BUILD_TYPE=Release
+    cmake --build build --target install -j14
+
+# Build and package the Cesium for Unreal plugin
+
+Store the absolute path of the root directory of the `cesium-unreal` repo in an environment variable for ease of reference:
+
+    export CESIUM_FOR_UNREAL_DIR=~/dev/cesium-unreal # adjust path for your system
+
+And build the plugin:
+
+    cd $UNREAL_ENGINE_DIR/Engine/Build/BatchFiles
+    ./RunUAT.sh BuildPlugin -Plugin="$CESIUM_FOR_UNREAL_DIR/CesiumForUnreal.uplugin" -Package="$CESIUM_FOR_UNREAL_DIR/../packages/CesiumForUnreal" -CreateSubFolder -TargetPlatforms=Linux
+
+And finally copy the built plugin into the Engine plugins directory:
+
+    mkdir -p $UNREAL_ENGINE_DIR/Engine/Plugins/Marketplace
+    cp -r $CESIUM_FOR_UNREAL_DIR/../packages/CesiumForUnreal $UNREAL_ENGINE_DIR/Engine/Plugins/Marketplace/
 
-# Creating the project files for the Unreal Engine game/project and the Cesium for Unreal plugin
+> Note: On Linux (unlike Windows), it is essential that the `CesiumForUnreal` plugin go in the `Plugins/Marketplace/` subdirectory, rather than directly in `Plugins/`. Otherwise, the relative paths to other plugin `.so` files that the Unreal Build Tool has built into the plugin will not resolve correctly.
 
-Instructions coming soon. On Linux, projects should probably be built from the command-line by using RunUAT.
+You should now be able to launch the Unreal Editor and open the `CesiumForUnrealSamples.uproject` found in `~/dev/cesium-unreal-samples`.
```

Comment:
If you agree with the above comment, then lets create a separate heading for this.